packages/icon-library/package.json

```diff
@@ -10,6 +10,7 @@
   "files": [
     "dist"
   ],
+  "types": "dist/index.d.ts",
```

Comment:
There is another PR that already adds this missing line.

Reply:
Duplicate here: https://github.com/Royal-Navy/standards-toolkit/pull/280/files

Reply:
Ah, must have cherry picked 2 commits by mistake